_posts/patterns/2016-03-01-dropdown-menu.md

```diff
@@ -0,0 +1,54 @@
+---
+layout:                 pattern
+title:                  Dropdown Menu
+date:                   2016-02-29 00:00:00
+
+categories:             patterns
+tags:
+- atomic
+- pattern
+- js
+- dropdown
+- menu
+
+slug:                   dropdown menu
+
+url_styles:             "patterns/_dropdown-menu.scss"
+
+description:            Dropdown Menu in a view.
+
+info:                   A JavaScript controlled dropdown menu.
+ui-toolkit-component:   dropdown-menu
+js:                     "/examples/dropdown-menu-js.html"
+---
+<h4 class="hd-6 example-set-hd">Basic Dropdown</h4>
+<div class="example-set">
+	<div class="js-user-cta"></div>
+</div>
+
+<h4 class="hd-6 example-set-hd">Dropdown with Image</h4>
+<div class="example-set">
+	<div class="js-user-cta-img"></div>
+</div>
+
+<h4 class="hd-6 example-set-hd">Already Rendered DOM</h4>
```

Comment:
I love having this example, btw. It makes it very clear how to do both approaches to rendering the dropdown.